test.js

```diff
@@ -293,11 +293,9 @@ test('.add() - handle task throwing error', async t => {
 	const queue = new PQueue({concurrency: 1});
 
 	queue.add(() => 'sync 1');
-	queue.add(() => {
+	t.throws(queue.add(() => {
```

Comment:
This should use `t.throwsAsync`.

From the docs:

> Adds a sync or async task to the queue. Always returns a promise. - https://github.com/sindresorhus/p-queue#addfn-options

Reply:
Perhaps I've got an older test library, but I'm getting `t.throwsAsync is not a function`.